basic/.config/rofi/config.rasi

```diff
@@ -9,7 +9,7 @@ configuration{
     hide-scrollbar: true;
     display-drun: "   Apps ";
     display-run: "   Run ";
-    display-window: " 﩯  Window";
+    display-window: "󰕰  Window";
```

Comment:
Do we need to keep a space here?

Also are the other icons okay without any changes?